Makefile

```diff
@@ -11,8 +11,6 @@
 # make regenerate
 #   Recompile all the necessary protos
 #   (requires protoc in path)
-# make test-xcode[-NAME]:
```

Comment:
I think this all would still be valid on the branch since the Xcode project is still there.

Reply:
This comment applied to all the other xcode related bits in the Makefile.  Likely should keep those here just incase someone was using them since it is part of the 1.x behaviors/support.

Reply:
Got you! Should be fixed now